curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-12
 
 # --description--
 
-Turn the words `cat photos` located inside `p` element into a link using the same value for the `href` attribute as the link below the `p` element. The `p` element should show the same text in the browser, but the words `cat photos` should now be a link. Make sure to remove the `a` element with the text `cat photos` on the line below the `p` element.
+The phrase `cat photos` shows up twice on our webpage. Replace the words `cat photos` located inside the `p` element with the entire `a` element. The `p` element should show the same text in the browser, but the words `cat photos` should now be a link. Remove the `a` element with the text `cat photos` on the line below the `p` element.
```

Comment:
Here's what I might have done...

```suggestion
You can see the words `Cat Photos` next to the image now, they are underlined. That's your anchor or link, feel free to give it a click. In the text of your `p` element, turn the words `Cat Photos` in into a link that links to `https://freecatphotoapp.com`. When you are done, remove the old anchor tag and text below the paragraph.
```

Reply:
Maybe this?
```
You can see the words `cat photos` underlined next to the image in the preview. That's your link; feel free to give it a click. Turn the words `cat photos` in the text of your `p` element into a link to `https://freecatphotoapp.com`. When you are done, remove the old anchor tag on the line below the paragraph element.
```

Reply:
I changed my suggestion to a combination of what it was and your suggestion @jeremylt - what do you think?

Reply:
The only thing I'd change is the capitalization of `cat photos` - If I recall correctly, its all lowercase in the challenge?

Reply:
Yup, you're right. I updated the suggestion.

Reply:
Sounds good 👍